### PR TITLE
added button components

### PR DESCRIPTION
**Pull request by PeeyushPrashant**

### CODE REVIEW

File: Components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.main-container > button {
+  align-items: center;
```
Comment (anuj-kumary) [inline Components/Button/button.css:2]: Don't target or give style to Html Element give class name and then use.

Comment (PeeyushPrashant) [reply Components/Button/button.css:2]: Done

**Review by anuj-kumary — APPROVED** [review]

**Review by varnita619 — APPROVED** [review]
Looks good to go

**Review by vinc3nati — APPROVED** [review]
Changes are good!

**Pull request merged**